src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -156,7 +156,7 @@
       AutomationProperties.AutomationId="Project_Button_Update"
       Click="InstallButton_Clicked"
       Visibility="{Binding IsInstalledVersionTopLevel, Converter={StaticResource BooleanToVisibilityConverter}, Mode=OneWay}"
-      IsEnabled="{Binding IsSelectedVersionInstalled, Converter={StaticResource NotNullOrTrueToBooleanConverter}}"
+      IsEnabled="{Binding IsInstallorUpdateButtonEnabled, Mode=OneWay}"
```

Comment:
This matches the `IsEnabled` in the install button above.

Reply:
Yes, the buttons works the same just the label changes. Before Floating feature, Install button was always enabled but now we need to make sure the versions selected is correct, this is the same for the Update Button.